mephisto/client/cli_commands.py

```diff
@@ -154,8 +154,11 @@ def get_wut_arguments(args):
                 checking_args[arg]["required"] = "[b]{requiredVal}[/b]".format(
                     requiredVal=checking_args[arg]["required"]
                 )
+            if arg == "tips_location":
+                checking_args[arg]["default"] = "path_to_task/assets/tips.csv"
```

Comment:
would be nice to leave a brief comment here explaining why we had to do this, for people down the road looking at this